crates/re_space_view_spatial/src/ui_3d.rs

```diff
@@ -380,6 +380,9 @@ pub fn view_3d(
             axis_length,
             re_renderer::OutlineMaskPreference::NONE,
         );
+
+        // If we are showing the axes for the space, then add the space origin to the bounding box.
+        state.scene_bbox.extend(glam::Vec3::ZERO);
```

Comment:
I'm not sure about this. Maybe. The bounding box is used for auto-positioning of the camera, and I'm not sure users are always interested in seeing the bounding box axes (though, to be fair, they did turned them on).

When we have a 3D grid we definitely don't want to include it in the bounding box (since it is infinite) 😆

Reply:
My thought is that since origin axes are opt-in, if you've taken the action of enabling it, you should see it when you reset your view without needing to do significant zooming / panning to "search" for it.